### DISCUSSION TITLE
Edit image source in editor

### DISCUSSION THREAD

**Discussion by Arbiter313** [notaapp/talk#373] [Feature Requests] [answered]
It would be great to be able to view and edit the source link for images when they are selected in the editor, without having to switch to the 'show markdown' mode. 
I believe this is a feature that was present in the Caret app. Obsidian also features this as seen below: 

<img width="471" alt="Screenshot 2023-02-21 at 1 06 55 am" src="https://user-images.githubusercontent.com/45447185/220222638-0ef2c51b-8e7f-4c3e-b5f9-7cd3ca04d173.png">

**Comment by astoilkov** [comment, marked as answer]
Yeah, I totally agree. We are planning to improve this in future releases.
